packages/core/http/core-http-router-server-internal/index.ts

```diff
@@ -7,6 +7,8 @@
  */
 
 export { filterHeaders } from './src/headers';
+export { versionHandlerResolvers } from './src/versioned_router';
```

Comment:
Not used in this PR, just exporting in preparation for future PR, happy to hold off on this change